server/src/main/java/org/apache/iotdb/db/conf/IoTDBDescriptor.java

```diff
@@ -863,6 +865,8 @@ private void loadUDFProps(Properties properties) {
           Integer.parseInt(initialByteArrayLengthForMemoryControl));
     }
 
+    conf.setExtDir(properties.getProperty("ext_root_dir", conf.getExtDir()));
```

Comment:
Add this property and its description to `src/assembly/resources/conf/iotdb-engine.properties`? :D

Reply:
done